diff -ruN ../nethack-3.4.2/include/config.h ./include/config.h
--- ../nethack-3.4.2/include/config.h	2003-08-30 00:07:15.000000000 +0000
+++ ./include/config.h	2015-11-26 23:43:11.782775835 +0000
@@ -183,7 +183,7 @@
  *	a tar-like file, thus making a neater installation.  See *conf.h
  *	for detailed configuration.
  */
-/* #define DLB */	/* not supported on all platforms */
+#define DLB	/* not supported on all platforms */
 
 /*
  *	Defining INSURANCE slows down level changes, but allows games that
diff -ruN ../nethack-3.4.2/src/Makefile ./src/Makefile
--- ../nethack-3.4.2/src/Makefile	2015-11-26 21:51:16.893284683 +0000
+++ ./src/Makefile	2015-11-27 15:41:31.937939674 +0000
@@ -230,8 +230,8 @@
 # WINTTYLIB = -ltermcap
 # WINTTYLIB = -lcurses
 # WINTTYLIB = -lcurses16
-# WINTTYLIB = -lncurses
-WINTTYLIB = -ltermlib
+WINTTYLIB = -lncurses
+#WINTTYLIB = -ltermlib
 #
 # libraries for X11
 # If USE_XPM is defined in config.h, you will also need -lXpm here.
diff -ruN ../nethack-3.4.2/win/tty/termcap.c ./win/tty/termcap.c
--- ../nethack-3.4.2/win/tty/termcap.c	2003-08-30 00:08:25.000000000 +0000
+++ ./win/tty/termcap.c	2015-11-26 20:46:09.108556866 +0000
@@ -835,9 +835,9 @@
 
 #include <curses.h>
 
-#ifndef LINUX
-extern char *tparm();
-#endif
+//#ifndef LINUX
+//extern char *tparm();
+//#endif
 
 #  ifdef COLOR_BLACK	/* trust include file */
 #undef COLOR_BLACK
